doc/nrf/protocols/bt/ble/conn_evt_cb.rst

```diff
@@ -0,0 +1,39 @@
+.. _ug_bt_conn_evt_cb:
+
+Synchronize data sampling with Bluetooth connection events
+##########################################################
+
+The Bluetooth connection event callback feature is used to inform the application about connection events.
+The connection event prepare callback triggered right before of a Bluetooth connection event.
+This allows the application to trigger sensor data collection for transmission during the upcoming connection event.
+
+.. figure:: images/ConnEvtCb.svg
+   :alt: Connection event prepare callback
+
+   Connection event prepare callback is called before a connection event
+
+The timing of the prepare callback is configured with the parameter `prepare_time_us`.
+The parameter must be set sufficiently large to allow the Bluetooth Stack to enqueue a data packet for the upcoming connection event.
+
+The :ref:`ble_conn_evt_cb` sample application demonstrates how this feature is used to reduce the latency between data sampling and data transmission.
+
+This feature can only be used with the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` and only when the Bluetooth Controller and application are located on the same core.
+
+Connection event callback with peripheral latency
+*************************************************
+
+When a connection is configured with peripheral latency, the peripheral device may sleep for `peripheral_latency` consecutive connection events to save power.
```

Comment:
```suggestion
When a connection is configured with peripheral latency, the peripheral device may sleep for ``peripheral_latency`` consecutive connection events to save power.
```
How is the peripheral latency configured (using a Kconfig option or something else?)

Reply:
The peripheral latency is configured run-time. It may change over the life-time of a connection. The central sets the initial value. The central may at any point in time change the value. The peripheral may request the central to change it